### DISCUSSION TITLE
Hexagon Croner

### DISCUSSION THREAD

**Discussion by mbstalin** [Hexagon/croner#167] [Q&A] [answered]
// Prepare an error handler
const errorHandler = (e) => {
	console.error(e);
};

// Start a job firing every second
const job = new Cron("* * * * * *", { catch: errorHandler }, (job) => {
	console.log('This will print!');
	throw new Error("This will be catched and printed by the error handler");
	console.log('This will not print, but the job will keep on triggering');
});

For the above code if we get an error in above CRONER Does it will start back or will be in stopped?

**Comment by Hexagon** [comment, marked as answer]
It will keep on running 👍 

If you do not provide catch: option, it would have stopped
